### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rekverr**
- [DEMO LINK](https://rekverr.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://rekverr.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,91 @@
...
+  padding-top: 20px;
+}
+
+.form {
+  position: relative;
+  margin-right: 8px;
+  margin-left: 8px;
+}
+
+.form [class='big-input'] {
```
Comment (volodymyr-soltys97) [inline src/style.css:20]: No need to use attributes as css selector, fix it everywhere
```suggestion
.big-input {
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,91 @@
...
+  padding-left: 62px;
+}
+
+.form [class='small-input'] {
+  padding-left: 33px;
+}
+
+.big-input {
+  font-family: 'Avenir M', sans-serif;
+  box-sizing: border-box;
```
Comment (volodymyr-soltys97) [inline src/style.css:30]: Move box-sizing to the * selector

File: src/style.css [hunk 3]
```diff
@@ -1 +1,91 @@
...
+body {
+  margin: 0;
+  padding-top: 20px;
+}
+
+.form {
+  position: relative;
+  margin-right: 8px;
+  margin-left: 8px;
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:18]: Redundant
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,91 @@
...
+@font-face {
+  font-family: 'Avenir M';
+  src:
+    local('Avemir'),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf) format(avenir);
+}
+
+body {
+  margin: 0;
```
Comment (volodymyr-soltys97) [inline src/style.css:10]: No need to reset margin for the body

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Comment by rekverr** [comment]
поправив те що Ви просили, але видає помилку при автоматичних тестах в mate 
image

**Pull request closed without merging**